packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -69,26 +69,23 @@ $ms-Persona-presenceSizeXl: 28px;
   line-height: 1;
   position: relative;
```

Comment:
Now that we're moving to a flexbox layout, do we still need position:relative? I assume it's a leftover from when something else that's position:absolute was positioned against this element?